avocado/utils/lv_utils.py

```diff
@@ -38,20 +38,21 @@ class LVException(Exception):
 
 def get_diskspace(disk):
```

Comment:
Shouldn't this be changed to `disks`?

Reply:
@harish-24 Thanks for it... missed it while copying..